src/lv_conf.h

```diff
@@ -230,7 +230,7 @@
  *-----------*/
 
 /*Enable the log module*/
-#define LV_USE_LOG 0
+#define LV_USE_LOG 1
```

Comment:
intentional? might be fine at the current loglevel

Reply:
Yes, i think at the moment (when we're deving like we are now) this makes more sense.